lib/routes/autocentre/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Data, Route } from '@/types';
+import parser from '@/utils/rss-parser';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '',
+    name: 'Автоцентр.ua',
+    maintainers: ['driversti'],
+    example: 'autocentre',
+    handler,
+};
+
+const createItem = (item) => ({
+    title: item.title,
+    link: item.link,
+    description: item.contentSnippet,
+});
+
+const getFromCacheOrNew = (item) => cache.tryGet(item.link, () => createItem(item));
```

Comment:
Don't use cache if it doesn't involve additional requests

Reply:
Done